src/generators/schema/website.php

```diff
@@ -142,11 +142,15 @@ private function internal_search_section( $data, Meta_Tags_Context $context ) {
 		 */
 		$search_url = \apply_filters( 'wpseo_json_ld_search_url', $context->site_url . '?s={search_term_string}' );
 
-		$data['potentialAction'] = [
+		/**
```

Comment:
The `/**` syntax with double asterisk is reserved for doc blocks. Multi line comments use only one asterisk `/*`.  All comments should end with a full stop. Regardless, I'd just remove this comment :) Similar code in WebPage and Article is not commented.